.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+
+on:
+  push:
+  release:
+    types: [published]
+
+env:
+  TOIT_VERSION: v2.0.0-alpha.121
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        # Zlib in Toit is not available on these platforms yet.
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Setup Toit
+        id: setup-toit
+        uses: toitlang/action-setup@v1
+        with:
+          toit-version: ${{ env.TOIT_VERSION }}
+
+      # Fetch the dependencies. Different for each platform.
+      - name: Install dependencies - Linux
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt-get update
+          sudo apt-get install ninja-build
```

Comment:
Since you don't need to build executables, I would avoid the ninja-part.
See https://github.com/toitlang/pkg-fs.